doc/tutorials/introduction/android_binary_package/O4A_SDK.rst

```diff
@@ -157,10 +157,10 @@ Get the OpenCV4Android SDK
 
    .. code-block:: bash
 
-      unzip ~/Downloads/OpenCV-2.4.10-android-sdk.zip
+      unzip ~/Downloads/OpenCV-2.4.11-android-sdk.zip
 
-.. |opencv_android_bin_pack| replace:: :file:`OpenCV-2.4.10-android-sdk.zip`
-.. _opencv_android_bin_pack_url: http://sourceforge.net/projects/opencvlibrary/files/opencv-android/2.4.10/OpenCV-2.4.10-android-sdk.zip/download
+.. |opencv_android_bin_pack| replace:: :file:`OpenCV-2.4.11-android-sdk.zip`
+.. _opencv_android_bin_pack_url: http://sourceforge.net/projects/opencvlibrary/files/opencv-android/2.4.10/OpenCV-2.4.11-android-sdk.zip/download
```

Comment:
shouldn't it be:
[...]opencv-android/**2.4.11**/OpenCV-2.4.11-android-sdk.zip/download